web/lib/src/dom/wasm_js_api.dart

```diff
@@ -73,6 +73,7 @@ extension type ModuleImportDescriptor._(JSObject _) implements JSObject {
   external ImportExportKind get kind;
   external set kind(ImportExportKind value);
 }
+@JS('WebAssembly.Module')
```

Comment:
Need to put an entry in the `pkg:web` changelog, too!

Reply:
Done. Added a bit more condensed description of changes for `pkg:web` changelog,

```Adding a `JS(LegacyNamespace.$extensionTypeName)` annotation on generated `JS` 
  objects if they have an IDL extended attribute `[LegacyNamespace=Foo]` defined 
  in their IDL description.```